website/docs/components/table/partials/content/link-usage.md

```diff
@@ -0,0 +1,24 @@
+## Links within cells
+
+!!! Info
+
+This guidance is an extension of the [Inline Link](/components/link/inline) guidelines.
+!!!
+
+Within a table, use `secondary` (`Foreground / Strong`) links as the default.
+
+Use `Body / 200 / Link` as the default typographic style within a table. This style increases the prominence a small amount to differentiate it from other string and non-interactive content.
```

Comment:
```suggestion
Use `Body / 200 / Link` as the default typographic style within a table. This style increases the prominence slightly to differentiate it from other string and non-interactive content.
```

Reply:
Also maybe change "string and non-interactive content" to "text content"?

Reply:
(I would normally not use the word "string" in a design context but reserve it more for contexts related to code.)